### PR TITLE
TELCODOCS-1742:Known issues from the Telco Core team

### PR DESCRIPTION
**Pull request by rohennes**
[TELCODOCS-1742](https://issues.redhat.com//browse/TELCODOCS-1742): Known issues from the Telco Core team

Version(s):
4.16

Issue: https://issues.redhat.com/browse/TELCODOCS-1745

Link to docs preview:
https://76998--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-known-issues_release-notes

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by ffromani — COMMENTED** [review]
/lgtm

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -705,6 +707,8 @@ The MetalLB `AddressPool` custom resource definition (CRD) has been deprecated f
 [id="ocp-4-16-scalability-and-performance-bug-fixes_{context}"]
 ==== Scalability and performance
 
+* Previously, the NUMA Resources Operator did not correctly handle `KubeletConfig` objects featuring an empty payload. As a consequence, the NUMA resources controller reported a `CrashLoopBackOff` status when applying a `NUMAResourcesOperator` resource in this scenario. With this release, improved filtering logic ensures that the NUMA Resources Operator handles `KubeletConfig` objects with empty payloads as expected. (link:https://issues.redhat.com/browse/OCPBUGS-30236[*OCPBUGS-30236*])
+
```
Comment (rohennes) [inline release_notes/ocp-4-16-release-notes.adoc:711]: @ffromani - can you review this bug fix text please?

File: release_notes/ocp-4-16-release-notes.adoc [hunk 2]
```diff
@@ -1207,6 +1211,8 @@ In the following tables, features are marked with the following statuses:
 
 [id="ocp-telco-core-4-16-known-issues_{context}"]
 
+* When a worker node's Topology Manager policy is changed, the NUMA-aware secondary pod scheduler does not respect this change, which can result in incorrect scheduling decisions and unexpected topology affinity errors. As a workaround, restart the NUMA-aware scheduler by deleting the NUMA-aware scheduler pod.
```
Comment (rohennes) [inline release_notes/ocp-4-16-release-notes.adoc:1214]: @ffromani - can you review this known issue text please?

File: release_notes/ocp-4-16-release-notes.adoc (hunk 1 above)
Comment (ffromani) [reply release_notes/ocp-4-16-release-notes.adoc:713]: LGTM

File: release_notes/ocp-4-16-release-notes.adoc (hunk 2 above)
Comment (ffromani) [reply release_notes/ocp-4-16-release-notes.adoc]: LGTM

File: release_notes/ocp-4-16-release-notes.adoc [hunk 3]
```diff
@@ -681,10 +681,16 @@ The MetalLB `AddressPool` custom resource definition (CRD) has been deprecated f
...
+
 [discrete]
 [id="ocp-4-16-node-tuning-operator-bug-fixes_{context}"]
 ==== Node Tuning Operator (NTO)
 
+* Previously, the Performance Profile Creator (PPC) incorrectly populated the `metadata.ownerReferences.uid` field for Day 0 performance profile manifests. As a consequence, it was not possible to apply a performance profile at Day 0 without manual intervention. With this release, the PPC does not generate the `metadata.ownerReferences.uid` field for Day 0 manifests. As a result, you can apply a performance profile manifest at Day 0 as expected. (link:https://issues.redhat.com/browse/OCPBUGS-29751[*OCPBUGS-29751*])
+
+* Previously, the `tuned` and `irqbalanced` daemons modified the Interrupt Request (IRQ) CPU affinity configuration. As a consequence, conflicts in the IRQ CPU affinity configuration might cause unexpected behaviour after a {sno} node restart. With this release, only the `irqbalanced` daemon determines IRQ CPU affinity configuration.(link:https://issues.redhat.com/browse/OCPBUGS-26400[*OCPBUGS-26400*])
```
Comment (rohennes) [inline release_notes/ocp-4-16-release-notes.adoc:692]: @yanirq - can you please review this bug fix release note text for OCPBUGS-26400? Thanks!

File: release_notes/ocp-4-16-release-notes.adoc [hunk 4]
```diff
@@ -681,10 +681,18 @@ The MetalLB `AddressPool` custom resource definition (CRD) has been deprecated f
...
+
+* Previously, when restarting a {sno} cluster with a performance profile applied, a race condition caused systemd processes to remain in the root directory instead of being moved to `/sys/fs/cgroup/cpuset/system.slice`. With this release, systemd processes are moved to `/sys/fs/cgroup/cpuset/system.slice` as expected following a {sno} node restart. (link:https://issues.redhat.com/browse/OCPBUGS-31692[*OCPBUGS-31692*])
```
Comment (rohennes) [inline release_notes/ocp-4-16-release-notes.adoc:694]: @rbaturov - can you please review this 4.16 bug fix release note? Thanks!

Comment (rbaturov) [reply release_notes/ocp-4-16-release-notes.adoc:694]: Looks good to me.

File: release_notes/ocp-4-16-release-notes.adoc (hunk 3 above)
Comment (yanirq) [reply release_notes/ocp-4-16-release-notes.adoc:690]: LGTM

File: release_notes/ocp-4-16-release-notes.adoc [hunk 5]
```diff
@@ -1207,6 +1215,16 @@ In the following tables, features are marked with the following statuses:
...
+
+* After applying a `SriovNetworkNodePolicy` resource, the CA certificate might be replaced during SR-IOV Network Operator webhook reconciliation. As a consequence, you might see `unknown authority` errors when applying SR-IOV Network node policies. As a workaround, try to re-apply the failed policies. (link:https://issues.redhat.com/browse/OCPBUGS-32139[*OCPBUGS-32139*])
+
+* If you delete a `SriovNetworkNodePolicy` resource for a virtual function with a `vfio-pci` driver type, the SR-IOV Network Operator is unable to reconcile the policy. As a consequence the `sriov-device-plugin` pod enters a continuous restart loop. As a workaround, delete all remaining policies affecting the physical function, then re-create them. (link:https://issues.redhat.com/browse/OCPBUGS-34934[*OCPBUGS-34934*])
```
Comment (rohennes) [inline release_notes/ocp-4-16-release-notes.adoc:1226]: @cgoncalves - can you please reveiw this known issue release note?

File: release_notes/ocp-4-16-release-notes.adoc [hunk 6]
```diff
@@ -1207,6 +1215,16 @@ In the following tables, features are marked with the following statuses:
...
++
+This issue affects CPU load balancing features because these features depend on the CPU Manager releasing CPUs to the available pool. Consequently, non-guaranteed pods might run with a reduced number of CPUs. As a workaround, schedule a pod with a `best-effort` CPU Manager policy on the affected node. This pod will be the last admitted pod and this ensures the resources will be correctly released to the available pool.(link:https://issues.redhat.com/browse/OCPBUGS-17792[*OCPBUGS-17792*])
+
+* After applying a `SriovNetworkNodePolicy` resource, the CA certificate might be replaced during SR-IOV Network Operator webhook reconciliation. As a consequence, you might see `unknown authority` errors when applying SR-IOV Network node policies. As a workaround, try to re-apply the failed policies. (link:https://issues.redhat.com/browse/OCPBUGS-32139[*OCPBUGS-32139*])
```
Comment (rohennes) [inline release_notes/ocp-4-16-release-notes.adoc:1224]: @cgoncalves - Can you please review this known issue release note -thanks

Comment (cgoncalves) [reply release_notes/ocp-4-16-release-notes.adoc:1224]: LGTM.
@zeeke WDYT?

File: release_notes/ocp-4-16-release-notes.adoc (hunk 5 above)
Comment (cgoncalves) [reply release_notes/ocp-4-16-release-notes.adoc:1226]: LGTM.
@zeeke WDYT?

Comment (zeeke) [reply release_notes/ocp-4-16-release-notes.adoc:1226]: Looks great

File: release_notes/ocp-4-16-release-notes.adoc (hunk 6 above)
Comment (zeeke) [reply release_notes/ocp-4-16-release-notes.adoc:1224]: LGTM

File: release_notes/ocp-4-16-release-notes.adoc [hunk 7]
```diff
@@ -1207,6 +1207,16 @@ In the following tables, features are marked with the following statuses:
 
 [id="ocp-telco-core-4-16-known-issues_{context}"]
 
+* When a worker node's Topology Manager policy is changed, the NUMA-aware secondary pod scheduler does not respect this change, which can result in incorrect scheduling decisions and unexpected topology affinity errors. As a workaround, restart the NUMA-aware scheduler by deleting the NUMA-aware scheduler pod. (link:https://issues.redhat.com/browse/OCPBUGS-34583[*OCPBUGS-34583*])
+
+* Due to an issue with Kubernetes, the CPU Manager is unable to return CPU resources from the last pod admitted to a node to the pool of available CPU resources. These resources are allocatable if a subsequent pod is admitted to the node. However, this in turn becomes the last pod, and again, the CPU manager cannot return this pod's resources to the available pool.
```
Comment (mburke5678) [inline release_notes/ocp-4-16-release-notes.adoc:1212]: Is there a bug to link to?

Comment (mburke5678) [reply release_notes/ocp-4-16-release-notes.adoc:1212]: I find this one a bit confusing, and a touch convoluted. But, I have no idea how to reword it. 

```suggestion
* Due to an issue with Kubernetes, the CPU Manager is unable to return to the pool of available CPU resources the resources from the last pod admitted to a node. The resources from the last pod are reallocatable if a subsequent pod is admitted to the node. However, this pod becomes the last pod, and again, the CPU manager cannot return this pod's resources to the available pool.
```

File: release_notes/ocp-4-16-release-notes.adoc [hunk 8]
```diff
@@ -1207,6 +1207,16 @@ In the following tables, features are marked with the following statuses:
...
++
+This issue affects CPU load balancing features because these features depend on the CPU Manager releasing CPUs to the available pool. Consequently, non-guaranteed pods might run with a reduced number of CPUs. As a workaround, schedule a pod with a `best-effort` CPU Manager policy on the affected node. This pod will be the last admitted pod and this ensures the resources will be correctly released to the available pool.(link:https://issues.redhat.com/browse/OCPBUGS-17792[*OCPBUGS-17792*])
```
Comment (mburke5678) [inline release_notes/ocp-4-16-release-notes.adoc:1214]:
```suggestion
This issue affects CPU load balancing features, which depend on the CPU Manager releasing CPUs to the available pool. Consequently, non-guaranteed pods might run with a reduced number of CPUs. As a workaround, schedule a pod with a `best-effort` CPU Manager policy on the affected node. This pod will be the last admitted pod and this ensures the resources will be correctly released to the available pool.(link:https://issues.redhat.com/browse/OCPBUGS-17792[*OCPBUGS-17792*])
```

**Comment by mburke5678** [comment]
@rohennes A couple of comments. Otherwise LGTM

File: release_notes/ocp-4-16-release-notes.adoc (hunk 7 above)
Comment (rohennes) [reply release_notes/ocp-4-16-release-notes.adoc:1212]: It is a confusing one. I've made minor changes based on your feedback, thanks!

**Pull request merged**